JeMPI_Apps/JeMPI_LibMPI/src/main/java/org/jembi/jempi/libmpi/dgraph/DgraphQueries.java

```diff
@@ -317,8 +317,9 @@ private static String getSimpleSearchQueryFilters(
          if (!param.value().isEmpty()) {
             String fieldName = camelToSnake(param.fieldName());
             Integer distance = param.distance();
+            String value = param.value();
             if (distance == 0) {
-               gqlFilters.add("alloftext(" + recordType + "." + fieldName + ", $" + fieldName + ")");
+               gqlFilters.add("regexp(" + recordType + "." + fieldName + ", /^" + value + "$/i)");
```

Comment:
```suggestion
               gqlFilters.add("regexp(" + recordType + "." + fieldName + ", /" + value + "/i)");
```
In case we would want to search for a substring within the field value. Otherwise, looks good :)

Reply:
@marrouchi this is for the exact matching search option. I would agree if we used the same for the fuzzy search options